src/tokens/tokens.interfaces.ts

```diff
@@ -95,6 +104,32 @@ export class TokenPool {
   config?: any;
 }
 
+export class TokenApproval {
+  @ApiProperty()
+  @IsNotEmpty()
+  poolId: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  owner: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  operator: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  approved: boolean;
+
+  @ApiProperty({ description: requestIdDescription })
+  @IsOptional()
+  requestId?: string;
+
+  @ApiProperty()
+  @IsOptional()
+  data?: string;
+}
```

Comment:
I think we're going to want an optional `config` parameter here, similar to the one for "create pool". It should be a JSON object that is passed in to FireFly and then passed through unchanged to the connector. Probably won't be used in this connector for the moment, but for ERC20 it would be used to pass allowance, etc. Maybe just worth defining on the interface for reference.